external/fv3fit/fv3fit/reservoir/config.py

```diff
@@ -59,47 +59,46 @@ class BatchLinearRegressorHyperparameters:
 
 
 @dataclass
-class ReservoirTrainingConfig:
+class ReservoirTrainingConfig(Hyperparameters):
     """
+    input_variables: variables and additional features in time series
+    output_variables: time series variables, must be subset of input_variables
```

Comment:
Should the subset condition be validated?

Reply:
Yes, added.